packages/integrations/telegram/api/webhook.ts

```diff
@@ -144,12 +82,42 @@ const webhook = async (req: AppNextApiRequest, res: NextApiResponse) => {
     },
   });
 
+  if (message) {
+    const existingContact = await prisma.contact.findUnique({
+      where: {
+        id: `${message.from.id}`,
```

Comment:
here contact.id is set to telegram from'id
We should not mix external unknown id types with our own
That's why there is an externalId column